### ISSUE TITLE
Smooth animation

### ISSUE THREAD

**Issue by Ramink78** [lukakilic/concentric-watch-face#1]
Hello, I hope you're doing great
I've installed this watch face on my Galaxy Watch 6, but I think its rotation animation could be smoother. It feels a bit laggy.

**Comment by rodrigoswz** [comment]
Yeah same here on my TicWatch Pro 5.

Compared with Muviz Watch Faces Collections, this is very laggy.

**Comment by lukakilic** [comment]
Hi, since I use the Watch Face Format there is not much I can do about the animation being laggy... Watch Face Format is designed to be (battery) efficient, it doesn't include any executable code and is entirely rendered by the smartwatch itself. From all the ways I've tried so far this is the "smoothest" version!

**Comment by Ramink78** [comment]
Yeah, I think with Watch Face Format, the watch faces can't render more than 15 fps, so your work was great and it's the WFF's limitation.

**Comment by lukakilic** [comment]
Yes that's right! Animations are currently limited to 15fps... If I find a way to make it smoother I'll definitely update it. So yes, it's a limitation of WFF!
